### DISCUSSION TITLE
How to submit the fp tasks to more than 2 nodes

### DISCUSSION THREAD

**Discussion by cui-hq** [deepmodeling/dpgen#1077] [Q&A] [answered]
In machine.json, if I use ""machine_type": "Shell"" in fp, how can I submit 20 tasks to 2 nodes, I don't have  scheduling system like slurm or PBS,thank you very much

**Comment by AnguseZhang** [comment, marked as answer]
We don't support such circumstance. Since this should be solved by scheduling system.

**Comment by cui-hq** [comment]
thank you for your response，hope everything goes well with your work.发自我的荣耀手机-------- 原始邮件 --------发件人： Yuzhi Zhang ***@***.***>日期： 2022年12月15日周四 16:03收件人： deepmodeling/dpgen ***@***.***>抄送： cui-hq ***@***.***>, Author ***@***.***>主    题： Re: [deepmodeling/dpgen] How to submit the fp tasks to more than 2 nodes (Discussion #1077)
We don't support such circumstance. Since this should be solved by scheduling system.
